Makefile

```diff
@@ -178,24 +178,10 @@ ifneq ($(origin DOCKER_CMD),undefined)
 #       if there's a docker container, jump into it and run from there
 	$(DOCKER_CMD) make jstest
 else
-#        the 'dropdb' command may fail, hence the prefix minus-sign
-	-dropdb lsmb_test_api
-	perl -Ilib bin/ledgersmb-admin create \
-            $${PGUSER:-postgres}@$${PGHOST:-localhost}/$${PGDATABASE:-lsmb_test_api}#xyz
-	perl -Ilib -Iold/lib bin/ledgersmb-admin user create \
-            $${PGUSER:-postgres}@$${PGHOST:-localhost}/$${PGDATABASE:-lsmb_test_api}#xyz \
-				--username="$${UIUSER:-Jest}" \
-				--password="$${UIPASSWORD:-Tester}" \
-				--employeenumber=1 \
-				--country='Canada' \
-				--first_name="$${UIUSER:-Jest}" \
-				--last_name="$${UIPASSWORD:-Tester}" \
-				--permission='Full Permissions'
 	# Make sure API definition is current
 	./utils/devel/rebuild_api.sh
 	# Test API answer
-	LSMB_NEW_DB_API=lsmb_test_api ./node_modules/.bin/jest $(TESTS)
-	-dropdb lsmb_test_api
+	./node_modules/.bin/jest $(TESTS)
```

Comment:
I need them to stop depending on a database already set somewhere else so they can be rerun at will.
I already separated them in groups for that purpose.

Reply:
Adding a database for each changes the run time from 0.5 sec to 5.1 sec.

Reply:
Ok. Any idea what the impact will be once we write more tests? Can we write our tests inside a transcation which we can roll back (instead of committing), so we run it in the same database in parallel?

Reply:
- No other category but OpenAPI validation tests should be tied to the server or the database, so Jest tests should be well under 5 seconds each.
 - All Jest tests can run in parallel, using a specified percentage of available CPU
 - Jstest and Server tests are independent and can run in 2 different streams, as we are aiming to separate the Server and the UI.